__tests__/integration.test.ts

```diff
@@ -85,4 +90,24 @@ describe('Node host', () => {
   it.skip('can call a function from plugin with args', async () => {
     await nvim.command('e! nvimrc');
   });
+
+  it('spawns a child host if $NVIM_NODE_HOST_DEBUG is set', async () => {
+    const childHost = cp.spawn(
+      process.execPath,
+      [path.join(__dirname, '..', 'bin', 'cli.js')],
+      { env: { NVIM_NODE_HOST_DEBUG: 1 } }
+    );
+    return axios.get('http://127.0.0.1:9229/json/list').then(
+      res => {
+        childHost.kill();
+        const debuggerData = res.data[0];
+        expect(debuggerData.webSocketDebuggerUrl).toMatch(
+          'ws://127.0.0.1:9229'
+        );
+      },
+      () => {
+        childHost.kill();
```

Comment:
would this be considered a test failure? should this throw an exception?

Reply:
I assume that, if the test returns a rejected promise, then Jasmine knows that the test failed. This line must run whether the promise is fulfilled or rejected so I wanted to use `Promise.prototype.finally` for this but it is not supported in Node 8 unless I use promise polyfill (or Babel?).

Reply:
If I pass the error to the reject callback, I get `ECONREFUSED`. The resolve callback doesn't run because it gets killed too early. Need sleep or timeout.